.github/workflows/deploy-smart.yaml

```diff
@@ -3,11 +3,51 @@ on:
   push:
     branches:
       - smart-develop
-      - feat-smart-multitenancy
 jobs:
+  build-validate:
+    name: Build and validate
+    runs-on: ubuntu-18.04
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Use Node.js
+        uses: actions/setup-node@v1
+        with:
+          node-version: 12
+      - uses: actions/setup-java@v1
+        with:
+          java-version: 1.8
+      - name: Install dependencies
```

Comment:
Nit pick: Use the working directory keyword to replace `cd` and `cd..`. Something like: 

```
- name: Build Hapi validator
  run: mvn --batch-mode --update-snapshots --no-transfer-progress clean install
  working-directory: javaHapiValidatorLambda
```

Reply:
yeah, that's probably better. That part was not modified as part of this release and is only a minor improvement to readability so I'm not changing it now.

Plus, I'd rather change it first on mainline and then on smart mainline(as usuaI). I'll open a ticket to change it later.